src/main/java/org/neo4j/sql2cypher/Translator.java

```diff
@@ -173,6 +177,36 @@ ResultStatement statement(Select<?> x) {
 		return buildableStatement.build();
 	}
 
+	Statement statement(QOM.Insert<?> insert) {
+		var table = insert.$into();
+		// TODO handle if this resolves to something unexpectedly different
+		var node = (Node) this.resolveTableOrJoin(table);
+
+		var rows = insert.$values();
+		var columns = insert.$columns();
```

Comment:
There's also a third case: when `insert.$select() != null`. It should work well with `UNWIND` as well, I think?